tests/ImageTests.php

```diff
@@ -447,7 +460,12 @@ protected function output($file)
     public function setUp()
     {
         $dir = $this->output('');
-        `rm -rf $dir`;
+        
+        if (PHP_OS === 'WINNT') {
+            `rd /s /q "$dir"`;
```

Comment:
Yes, so just add it on `require-dev`. That would not be required on other project. :+1: